tests/scripts/AuthDataLoaderTests.js

```diff
@@ -0,0 +1,151 @@
+/*!
```

Comment:
"scripts" is a strange directory name for tests - don't we more typically use tests/js etc?